lib/cc/analyzer/issue_relative_path_validation.rb

```diff
@@ -11,6 +11,13 @@ def message
 
       private
 
+      def relative_to?(directory)
```

Comment:
I'm interested in having an in-chat conversation about how we have decided / are deciding to "address symlinks". I think there's some nuance there and I know we've fought this battle before.